api/KS.FiksProtokollValidator.WebAPI/TestCases/no.ks.fiks.politisk.behandling.klient.v1/SendOrienteringssakN1/testInformation.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+    "testName": "Send Orienteringssak",
+    "description": "Send Orienteringssak",
+    "testStep": "Normalsituasjon 1 Input Send Utvalg, retur kvittering på mottak",
+    "messageType": "no.ks.fiks.politisk.behandling.klient.sendorienteringssak.v1",
+    "operation": "SendOrienteringssak",
+    "situation": "N1",
+    "expectedResult": "En kvittering på mottak",
+    "queriesWithExpectedValues": [
+    ],
+    "supported": true,
+    "protocol": "no.ks.fiks.politisk.behandling.klient.v1"
+}
```

Comment:
Sjekk at alle TestCases filer kommer med ved bygging og publish.

Reply:
Ordnet slik at attachments kommer med for denne.